script.js

```diff
@@ -0,0 +1,31 @@
+const textsToCopyContainer = document.querySelector('#textsToTypeContainer')
+const paragraphs = []
+
+fetch('https://flipsum-ipsum.net/api/icw/v1/generate?ipsum=recipe-ipsum-text-generator&start_with_fixed=0&paragraphs=4').then((response) => {
+    return response.json()
+}).then((data) => {
+    data.forEach((textOutput) => {
+        //the line below replaces any accented character by the matching non-accented character
+        //e.g. "é" becomes "e"
+        textOutput = textOutput.normalize("NFD").replace(/\p{Diacritic}/gu, "")
+        paragraphWordsArray =  textOutput.split(' ')
+        paragraphs.push(paragraphWordsArray)
+    }) 
+    
+    const referenceTextArray = paragraphs[0].concat(paragraphs[1], paragraphs[2], paragraphs[3]) // for use in future stories
+    let count = 0
+    paragraphs.forEach((paragraph, indexParagraph) => {
+        let stringForHtml = '<p class="textParagraph" id="paragraph' + indexParagraph + '">'
```

Comment:
due to the new autoscrolling functionality, separate paragraphs will not be needed anymore and the logic can be simplified (as discussed & will also discuss tomorrow)

Reply:
Done: we decided to display all our words on the page as a big paragraph. In story 3 we plan to make a function that will find where line breaks are and scroll as the user starts typing a word belonging to the next line.